### ISSUE TITLE
Nimbus stopped syncing with Prater at block 250560

### ISSUE THREAD

**Issue by paulmillr** [status-im/nim-beacon-chain#2863]
**Describe the bug**

I've managed to sync the mainnet, then deleted the directory and started syncing prater (same with geth @ goerli). It was syncing until it stopped in one position, at block 250560.

**To Reproduce**
1. Platform details (OS, architecture): debian 11 @ raspi
2. Branch/commit used: `nimbus-eth2_Linux_arm64v8_20210830_7d1efa44.tar.gz` aka 7d1efa443d6051774d0283d49fcd87ce33a4681c
3. Commands being executed:

```
[Unit]
Description=Nimbus Eth2 Client
Wants=network-online.target
After=network-online.target
[Service]
User=nimbus
Group=nimbus
Type=simple
Restart=always
RestartSec=5
WorkingDirectory=/home/nimbus
Environment="ClientIP=17.254.215.180"
ExecStart=/usr/local/bin/nimbus_beacon_node --data-dir=/home/nimbus --web3-url=ws://127.0.0.1:8546 --nat=extip:17.254.215.180 --network=prater
[Install]
WantedBy=multi-user.target
```

4. Relevant log lines: https://gist.github.com/paulmillr/42adaaec07b901399e2eea896a468f6e

**Comment by tersec** [comment]
That's when Prater transitioned to Altair. Currently, only the `unstable` branch supports Altair.

**Comment by paulmillr** [comment]
at slot `250560`? How so? I've thought altair was around `1200000` which is a lot later.

**Comment by tersec** [comment]
https://github.com/status-im/nimbus-eth2/releases/tag/altair-beta -- If you're running a Prater node, this is an opportunity to test your infrastructure for the upcoming Altair upgrade -- Prater will be upgrading to Altair on 2021-09-02 12:24:00 UTC.

**Comment by paulmillr** [comment]
I understand that. I've used nightly build and thought it supports altair.

I've also thought the error won't happen until Altair-fork-block which is 900k blocks later. So this is 100% related to Altair? Not to something else?

**Comment by paulmillr** [comment]
Actually as i've mentioned in the top post, it's already 7d1efa44 which is the same commit as altair-beta.

**Comment by tersec** [comment]
The nightly build does support Altair.

https://github.com/eth2-clients/eth2-networks/blob/master/shared/prater/config.yaml shows the Prater altair fork epoch is 36,660, or slot 1,173,120, so you're correct this is likely something besides the Altair transition.

**Comment by paulmillr** [comment]
This is not `nightly` build! This is altair build. The commit here https://github.com/status-im/nimbus-eth2/releases/tag/altair-beta is 7d1efa443d6051774d0283d49fcd87ce33a4681c which is what i'm using.

**Comment by arnetheduck** [comment]
This is fixed in https://github.com/status-im/nimbus-eth2/pull/3189 and currently part of the `unstable` branch - will be included in the next release.

**Comment by arnetheduck** [comment]
Released in 1.6.0
